int32 TYPE_UKNOWN=0
int32 TYPE_NOSOLUTIONFORPLAN=1
int32 TYPE_ROBOTOUTOFMAP=2
int32 TYPE_MAPISEMPTY=3
int32 TYPE_FAILTEDTOCALLSERVICEC22=4
int32 TYPE_TARGETPOSITIONINCORRECT=5
int32 type
string description